python/dgl/subgraph.py

```diff
@@ -726,8 +747,11 @@ def khop_out_subgraph(graph, nodes, k, *, relabel_nodes=True, store_ids=True):
 
     Returns
     -------
-    G : DGLGraph
+    DGLGraph
         The subgraph.
+    Tensor or dict[str, Tensor], optional
```

Comment:
Is this a breaking change? Is it possible to store the inverse in the ndata so that the returns are the same for both cases?